doc/admin/deployment.md

```diff
@@ -68,10 +68,6 @@ $ sudo grub2-mkconfig --output=/boot/grub2/grub.cfg
 $ sudo reboot
 ```
 
-!!! warning
-    VFIO support is currently planned for DAOS 1.2 and won't work until
-    we move to SPDK 20.01.1
```

Comment:
Well, VFIO is not officially tested on 1.0, so i would still add a warning that this will be supported only 1.2.

Reply:
Agreed that we shouldn't land this to release/0.9, but this seems like it's only targeted for master. We could leave a warning that it's a new feature and may still have issues, but at this point it's expected to work on master.

Reply:
so shall we land it or leave it for the moment?

Reply:
How about the following text: 
VFIO support is a new feature for DAOS 1.2 and has been tested on the following platforms:
  * CentOS 7.7

I fear that this feature may be something of a moving target for us. Hopefully being on LTS releases of SPDK/DPDK will mean that those teams are responsive to breaking kernel changes so that we don't have a repeat of what happened when 7.7 was released.